### DISCUSSION TITLE
moments: Error 1406: Data too long for column 'album_slug' at row 1 (update folder)

### DISCUSSION THREAD

**Discussion by whoshoe** [photoprism/photoprism#2179] [Q&A] [answered]
Hi there,

I've seen a lot of this error in the log. After looking at a similar issue brought up by [DXorSX](https://github.com/DXorSX) https://github.com/photoprism/photoprism/discussions/2151 I checked the path of the update folder "/photoprism/originals/nextcloud" doesn't seem to be too long. What am I missing here? Is it a bug?

Thanks!

**Comment by lastzero** [comment, marked as answer]
See:
- https://github.com/photoprism/photoprism/issues/2181

Development Preview for testing will be available soon. The Docker image is currently being built.
